spec/brcobranca/retorno/cnab400/itau_spec.rb

```diff
@@ -0,0 +1,55 @@
+# -*- encoding: utf-8 -*-
+require 'spec_helper'
+
+RSpec.describe Brcobranca::Retorno::Cnab400::Itau do
+  before do
+    @arquivo = File.join(File.dirname(__FILE__), '..', '..', '..', 'arquivos', 'CNAB400ITAU.RET')
+  end
+
+  it 'Ignora primeira linha que é header' do pagamentos = described_class.load_lines(@arquivo)
+    pagamento = pagamentos.first
+    expect(pagamento.sequencial).to eql('000002')
+  end
+
+  it 'Transforma arquivo de retorno em objetos de retorno retornando somente as linhas de pagamentos de títulos sem registro' do
+    pagamentos = described_class.load_lines(@arquivo)
+    expect(pagamentos.size).to eq(53) # deve ignorar a primeira linha que é header
+    pagamento = pagamentos.first
+    expect(pagamento.agencia_com_dv).to eql('0730')
+    expect(pagamento.cedente_com_dv).to eql('035110')
+    expect(pagamento.nosso_numero).to eql('00000011')
+    expect(pagamento.carteira_variacao).to eql('109')
+    expect(pagamento.carteira).to eql('I')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.